.circleci/config.yml

```diff
@@ -97,6 +105,22 @@ jobs:
           paths:
             - ./*
 
+  install_cli:
+    executor: base-executor
+    working_directory: *cli_working_dir
+    steps:
+      - attach_workspace: *attach_options
+      - restore_cache: *cli_package_lock_key
+      - run: npm ci
```

Comment:
I must admit that I'm a bit unexperienced with this command. But doesn't this require to have `npm install` first? So this might work as long as CircleCI can restore a cache. But for the first time on a new cache version this would fail, wouldn't it? :thinking:

Reply:
This is a clean install. It deletes `node_modules` and then installs everything as it is in `package-lock.json`

https://docs.npmjs.com/cli/ci.html